lib/dialects/abstract/query.js

```diff
@@ -86,7 +86,10 @@ class AbstractQuery {
 
     sql = sql.replace(/\$(\$|\w+)/g, (match, key) => {
       if ('$' === key) {
-        return options.skipUnescape ? match : key;
+        if (!['postgres', 'sqlite'].includes(dialect)) {
+          return options.skipUnescape ? match : key;
+        } 
+        return match;
```

Comment:
Hello, just out of curiosity, why was this change necessary? (I'm a maintainer but I'm sure you understand this part much better than I do)

Reply:
Because $ is used to escape a literal $ sign. If we unescape `$$1`  with `$1` , then later we replace logSql with bind Parameter, the  `$1` will be `${Parameter}`, but actual expect `$1`.
 For example we have a query   
`sequelize.query('SELECT *, "text with literal $$1 and literal $$status" as t FROM projects WHERE status = $1',
  { bind: ['active'], type: sequelize.QueryTypes.SELECT }
)`  
the log sql will be  
`SELECT *, "text with literal 'active' and literal $status" as t FROM projects WHERE status = 'active'`
but expect   
`SELECT *, "text with literal $1 and literal $status" as t FROM projects WHERE status = 'active'`.